### PR TITLE
Added Lists to origin-version-changer

### PR DESCRIPTION
**Pull request by soltysh**
The changer didn't process List objects.

@smarterclayton @mfojtik PTAL

### CODE REVIEW

File: cmd/origin-version-change/main.go [hunk 1]
```diff
@@ -43,10 +43,18 @@ func isYAML(data []byte) bool {
 }
 
 func changeTemplateObjectsVersion(t *templateapi.Template) {
-	if errs := kruntime.DecodeList(t.Objects, api.Scheme); len(errs) > 0 {
+	changeObjects(t.Objects)
+}
+
+func changeListObjectsVersion(t *api.List) {
```
Comment (mfojtik) [inline cmd/origin-version-change/main.go]: i don't this there is a point to have this one line function... just call changeObjects()

**Comment by soltysh** [comment]
@mfojtik updated.

**Comment by mfojtik** [comment]
@soltysh is the travis just a random failure?

**Comment by mfojtik** [comment]
LGTM

**Comment by soltysh** [comment]
I guess so, I haven't change any internal origin part.
[test] for double check.

**Comment by soltysh** [comment]
Travis green.
[merge]

**Pull request merged**